typescript/looker/rtl/nodeTransport.ts

```diff
@@ -32,15 +32,15 @@ import {
   responseMode,
   ResponseMode,
   SDKResponse,
-  StatusCode, trace, Values,
+  StatusCode, trace, Values, IRequestHeaders,
 } from './transport'
 
 import rq, { Response, Request } from 'request'
 import rp from 'request-promise-native'
 import { PassThrough, Readable } from 'readable-stream'
 import { BaseTransport } from './baseTransport'
 
-type RequestOptions = rq.RequiredUriUrl & rp.RequestPromiseOptions
+export type RequestOptions = rq.RequiredUriUrl & rp.RequestPromiseOptions
```

Comment:
That specific export was introduced because the latest linter complains about the type not being directly visible to callers outside `nodeTransport.ts`. Excellent catch, though. Great attention to detail. That's another one I'd like @lukelooker to chime in on.